public/index.html

```diff
@@ -5,16 +5,15 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="./styles.css">
-    <link rel="stylesheet" href="https://i.icomoon.io/public/temp/1a05ea6e8c/UntitledProject/style.css">
-    <link rel="preconnect" href="https://fonts.gstatic.com"> 
+    <link rel="stylesheet" href="https://i.icomoon.io/public/temp/d2904d5eb4/UntitledProject/style.css">    <link rel="preconnect" href="https://fonts.gstatic.com"> 
     <link href="https://fonts.googleapis.com/css2?family=DM+Mono:ital,wght@0,300;0,400;0,500;1,300;1,400;1,500&display=swap" rel="stylesheet">
     <title>Board Smash</title>
 </head>
 <body>
     
     <header>
         <nav>
-            <div><a href="http://www.boardsmash.com"><img id="logo" src="./Assets/logo.svg" alt="Board Smash"></a></div>
+            <div><a href="http://www.boardsmash.com"><img id="logo" src="./assets/logo.svg" alt="Board Smash"></a></div>
             <div class="avatar">JD</div>
             <div id="tab1" class="tab"><span class="icomoon icon-type"></span> Type</div>
```

Comment:
**nitpick:** Ignore including numbers like `1,2,3...` in the naming, instead prefer letters like `one, two, three. It's more of the convention, and can be ignored if its not comfortable to you